KeePassKitTests/KPKTestTokenStream.m

```diff
@@ -81,14 +81,14 @@ - (void)testReferenceTokenizing {
 }
 
 - (void)testEmojiTokenizing {
-  KPKTokenStream *stream = [KPKTokenStream tokenStreamWithValue:@"{TAB}😀A👢B👴🏼C{🐱}{ENTER}"];
+  KPKTokenStream *stream = [KPKTokenStream tokenStreamWithValue:@"{TAB}😀A👢B👴C{🐱}{ENTER}"];
```

Comment:
This should not be necessary. The toke streams is a WIP and should work with composed character sequences so it should respect the toned emoji.

Reply:
https://github.com/MacPass/KeePassKit/blob/df21b5edd97e0677437dbc7457cc3fe9e4e55056/KeePassKit/Token/KPKTokenStream.m#L112 actually uses NSStringEnumerationByComposedCharacterSequences so this seems strange for me to happen or is this an issue with NSString API?

Reply:
Yes, that is strange indeed! I will revert that change and d more research.

Reply:
Seems like NSStringEnumerationByComposedCharacterSequences do not recognize composed emoji like 👴🏼`\U0001F474\U0001F3FB` on 32 bit arch.

While it works well with composed characters such as ḱ (`k\U00000301`) ṷ (`u\U0000032D` or even ṓ (`o\U00000304\U00000301`)!

I tried using NSData with UTF32 encoding with no success. 

I suggest to have differents tests on both arch:
```objective-c
- (void)testEmojiTokenizing {
#if __LP64__ || NS_BUILD_32_LIKE_64
  KPKTokenStream *stream = [KPKTokenStream tokenStreamWithValue:@"{TAB}😀A👢B👴🏼C{🐱}{ENTER}"];
  XCTAssertEqual(9, stream.tokenCount);
  XCTAssertEqualObjects(@"{TAB}", stream.tokens[0].value);
  XCTAssertEqualObjects(@"😀", stream.tokens[1].value);
  XCTAssertEqualObjects(@"A", stream.tokens[2].value);
  XCTAssertEqualObjects(@"👢", stream.tokens[3].value);
  XCTAssertEqualObjects(@"B", stream.tokens[4].value);
  XCTAssertEqualObjects(@"👴🏼", stream.tokens[5].value);
  XCTAssertEqualObjects(@"C", stream.tokens[6].value);
  XCTAssertEqualObjects(@"{🐱}", stream.tokens[7].value);
  XCTAssertEqualObjects(@"{ENTER}", stream.tokens[8].value);
#else
  KPKTokenStream *stream = [KPKTokenStream tokenStreamWithValue:@"{TAB}😀A👢B👴C{🐱}{ENTER}"];
  XCTAssertEqual(9, stream.tokenCount);
  XCTAssertEqualObjects(@"{TAB}", stream.tokens[0].value);
  XCTAssertEqualObjects(@"😀", stream.tokens[1].value);
  XCTAssertEqualObjects(@"A", stream.tokens[2].value);
  XCTAssertEqualObjects(@"👢", stream.tokens[3].value);
  XCTAssertEqualObjects(@"B", stream.tokens[4].value);
  XCTAssertEqualObjects(@"👴", stream.tokens[5].value);
  XCTAssertEqualObjects(@"C", stream.tokens[6].value);
  XCTAssertEqualObjects(@"{🐱}", stream.tokens[7].value);
  XCTAssertEqualObjects(@"{ENTER}", stream.tokens[8].value);
#endif
}
```